conf/appEvents.routes

```diff
@@ -1,7 +1,5 @@
 # microservice specific routes
 
-GET         /helloworld                       @uk.gov.hmrc.apiplatformevents.controllers.ApplicationEventsController.helloworld
-
 POST        /teamMemberAdded                  @uk.gov.hmrc.apiplatformevents.controllers.ApplicationEventsController.teamMemberAdded
 POST        /teamMemberRemoved                @uk.gov.hmrc.apiplatformevents.controllers.ApplicationEventsController.teamMemberRemoved
 POST        /clientSecretAdded                @uk.gov.hmrc.apiplatformevents.controllers.ApplicationEventsController.clientSecretAdded
```

Comment:
Clearly a bit late now, but really don't think we should have an endpoint-per-event-type...